packages/liveblocks-node/src/utils.ts

```diff
@@ -36,9 +47,15 @@ export function assertSecretKey(
   value: unknown,
   field: string
 ): asserts value is string {
-  if (!isNonEmpty(value) || !value.startsWith("sk_")) {
+  if (!startsWith(value, "sk_")) {
+    throw new Error(
+      `Invalid value for field '${field}'. Secret keys must start with 'sk_'. Please provide the secret key from your Liveblocks dashboard at https://liveblocks.io/dashboard/apikeys.`
+    );
+  }
+
+  if (!/^[\w-]+$/.test(value)) {
```

Comment:
NB: I will tend to put this RegExp in a const roughly named `SECRET_KEY_RG` and to add a bit of jsdoc to help developers understand what this RegExp represents and for what it's useful such as `Regular Expression to check if the secret key contains one or more word characters or hyphens`. 

It could be helpful for legacy and code intent knowledge sharing.

Reply:
Agreed, I'll put it in a constant. I normally do this as well. Not sure why I left it like this. Thanks for catching 👍  Fixed in e33f0d6ee2b171e1a4b1ad86c64f7dcf54295756.